android/widgets/src/main/java/org/nativescript/widgets/Async.java

```diff
@@ -296,10 +296,20 @@ public void addHeaders(HttpURLConnection connection)
 				{
 					return;
 				}
+				boolean hasAcceptHeader = false;
 
 				for (KeyValuePair pair : this.headers)
 				{
-					connection.addRequestProperty(pair.key.toString(), pair.value.toString());
+					String key = pair.key.toString();
+					connection.addRequestProperty(key, pair.value.toString());
+					if (key.contentEquals("Accept-Encoding")) {
```

Comment:
I just realized after looking up the RFC for headers; that the key is supposed to be case-insensitive.   I'll be doing another fix to make this case-insensitive check.

Reply:
This issue should now be resolved.  :grinning: